ci/pipelines/s3-resource-simple-template.yml

```diff
@@ -11,7 +11,7 @@ s3-resource-simple-git: &s3-resource-simple-git
 resource_types:
 
 - name: s3-resource-simple
-  type: docker-image
+  type: registry-image
```

Comment:
I will probably have to remove: insecure_registries: ['docker.internal:5000']

Reply:
Setting the `repository` key should not be needed any more, correct?